tests/TestGIN.py

```diff
@@ -1,104 +1,81 @@
 import random
-import sys
-import io
-
-sys.path.append("")
 import unittest
 
 import numpy as np
-import matplotlib.image as mpimg
-import matplotlib.pyplot as plt
 
 from causallearn.search.HiddenCausal.GIN.GIN import GIN
 
 
 class TestGIN(unittest.TestCase):
     def test_case1(self):
-        sample_size = 1000
-        np.random.seed(0)
-        L1 = np.random.uniform(-1, 1, size=sample_size) ** 5
-        L2 = np.random.uniform(0.5, 2.0) * L1 + np.random.uniform(-1, 1, size=sample_size) ** 5
-        X1 = np.random.uniform(0.5, 2.0) * L1 + np.random.uniform(-1, 1, size=sample_size) ** 5
-        X2 = np.random.uniform(0.5, 2.0) * L1 + np.random.uniform(-1, 1, size=sample_size) ** 5
-        X3 = np.random.uniform(0.5, 2.0) * L2 + np.random.uniform(-1, 1, size=sample_size) ** 5
-        X4 = np.random.uniform(0.5, 2.0) * L2 + np.random.uniform(-1, 1, size=sample_size) ** 5
-
+        sample_size = 500
+        random.seed(42)
+        np.random.seed(42)
+        L1 = np.random.uniform(-1, 1, size=sample_size)
+        L2 = np.random.uniform(1.2, 1.8) * L1 + np.random.uniform(-1, 1, size=sample_size)
+        X1 = np.random.uniform(1.2, 1.8) * L1 + 0.2 * np.random.uniform(-1, 1, size=sample_size)
+        X2 = np.random.uniform(1.2, 1.8) * L1 + 0.2 * np.random.uniform(-1, 1, size=sample_size)
+        X3 = np.random.uniform(1.2, 1.8) * L2 + 0.2 * np.random.uniform(-1, 1, size=sample_size)
+        X4 = np.random.uniform(1.2, 1.8) * L2 + 0.2 * np.random.uniform(-1, 1, size=sample_size)
         data = np.array([X1, X2, X3, X4]).T
         data = (data - np.mean(data, axis=0)) / np.std(data, axis=0)
-        g, k = GIN(data)
-        print(g, k)
+        _, k = GIN(data)
```

Comment:
please give better naming (instead of naming it "k")

Basically you should never use not-meaningful variable name like "k" (only exception is well-known i and j for loops)

Reply:
Thanks for the suggestion, I will update it later. I see that k is often used in papers to indicate causal order, e.g. [LiNGAM](https://www.jmlr.org/papers/v7/shimizu06a.html), so I just used k.

Reply:
hmmm, makes sense haha. But it would still be good to name it clearly in our codebase (you can add comment to say this variable corresponds to k in paper)